index.html

```diff
@@ -48,7 +48,189 @@ <h3 class="lets-connect">LET'S CONNECT</h3>
+                </div>
+             </div>
+            </div>
+             
+            
+
+
+                
+               <div class="project-container">
+                <div class="postcollection">
+                <div class="img-container"><img class="snapshoot-img" src="images/Snapshoot Portfolio (3).png" alt="snapshot portfolio 3"></div>
+                <div class="project-review">
+                <h3>Tonic</h3>
+                </div>
+                <div class="backend-dev">
+                    <div><p class="canopy">CANOPY</p></div>
+                    <div><img src="images/Counter (1).png" alt=""></div>
+                    <div><p class="backend">Back End Dev</p></div>
+                    <div><img src="images/Counter (1).png" alt=""></div>
+                    <div><p class="backend">2015</p></div>
+                </div>
+                <div>
+                    <p class="access-project">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="programming-lang">
+                    <ul>
+                    <li class="lang">html</li>
+                    <li class="lang">css</li>
+                    <li class="lang">javaScript</li>
+                    </ul>
+                </div>
+                <div>
+                    <button class="project-btn"><a href="#">See Project</a></button>
+                </div>
+                </div>
+            </div>
+    
+
+            <div class="project-container">
+                <div class="postcollection">
+                    <div class="img-container"><img class="snapshoot-img" src="images/Snapshoot Portfolio (4).png" alt="portfolio 2"></div>
+                    <div class="project-review">
+                        <h3>Multi Post Stories</h3>
+                    </div>
+                    <div class="backend-dev">
+                        <div><p class="canopy">CANOPY</p></div>
+                        <div><img src="images/Counter (1).png" alt=""></div>
+                        <div><p class="backend">Back End Dev</p></div>
+                        <div><img src="images/Counter (1).png" alt=""></div>
+                        <div><p class="backend">2015</p></div>
+                    </div>
+                    <div>
+                        <p class="access-project">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <div class="programming-lang">
+                        <ul>
+                        <li class="lang">html</li>
+                        <li class="lang">css</li>
+                        <li class="lang">javaScript</li>
+                        </ul>
+                    </div>
+                    <div>
+                        <button class="project-btn"> <a href="#">See Project</a></button>
+                    </div>
```

Comment:
- [ ] Kindly implement the display technology for CSS. The previous CR asked to change the project requirement to use Grid and flex. For the projects container properties ` display: Grid` and for project container properties  `display: flex`. Please fix it :wrench: